tests/meson.build

```diff
@@ -17,6 +17,7 @@ test_src = files(
     'test_Step.cc',
     'test_time_types.cc',
     'test_Timeout.cc',
+    'test_Timeout.cc',
```

Comment:
?

Reply:
Oops! Nervous finger by copy-and-paste :-). Removed.